src/foam/nanos/notification/email/EmailTemplateApplyEmailPropertyService.js

```diff
@@ -31,15 +31,20 @@ foam.CLASS({
         logger = new StdoutLogger();
       }
       if ( templateArgs == null ) {
+
         return emailMessage;
       }
       String templateName = (String)templateArgs.get("template");
-      if ( SafetyUtil.isEmpty(templateName) ) return emailMessage;
+      if ( SafetyUtil.isEmpty(templateName) ) {
+        logger.info("No email template name");
```

Comment:
with out more information i feel this logger is useless

Reply:
updated